data/VT/incentives.json

```diff
@@ -423,7 +441,8 @@
     "start_date": 2022,
     "end_date": 2023,
     "short_description": {
-      "en": "BED customers may receive a rebate up to $2,500 on qualifiying HPs."
+      "en": "BED customers may receive a rebate up to $2,500 on qualifiying HPs, plus an additional $500 for a second HP.",
```

Comment:
There are a few rebates where `qualifying` is misspelled. Will need to be changed probably in the spreadsheet, here, and in the test fixture.

Reply:
Thank you for catching that! Fixed.